week-2/implement/vat.js

```diff
@@ -8,3 +8,10 @@
 // Given a number,
 // When I call this function with a number
 // it returns the new price with VAT added on
+
+function Vat(num){
```

Comment:
function names should start with a lower case letter in JS.

Reply:
Alright. I will keep that in mind.